src/comp/comp.py

```diff
@@ -24,48 +28,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [str.name for str in humans if str.name[0].lower() == 'd']
```

Comment:
You don't have to `.lower()` this
```suggestion
a = [str.name for str in humans if str.name[0] == 'D']
```
You can just take out the extra step

Reply:
But actually I see that you are guarding against if the letter is lowercase initially so good job on that